src/generic/gridsel.cpp

```diff
@@ -48,13 +48,55 @@ wxGridSelection::wxGridSelection( wxGrid * grid,
 {
     m_grid = grid;
     m_selectionMode = sel;
+    m_isSelecting = false;
 }
 
 bool wxGridSelection::IsSelection()
 {
     return !m_selection.empty();
 }
 
+
+void wxGridSelection::StartSelecting()
+{
+    m_isSelecting = true;
+}
+void wxGridSelection::EndSelecting()
+{
+    m_isSelecting = false;
+    //send the RANGE_SELECTED events
+    if (IsSelection())
+    {
+        size_t n;
+        wxRect r;
+        wxGridCellCoords coords1, coords2;
+
+        // deselect all blocks and update the screen
```

Comment:
This comment seems to have been wrongly copy-and-pasted:
```suggestion
        // Send selection events for all the selected blocks.
```